formats/mainstream_browse_page/publisher/details.json

```diff
@@ -3,6 +3,9 @@
   "type": "object",
   "additionalProperties": false,
   "properties": {
+    "child_ordering": {
```

Comment:
This field will need to be present in the content-items for both top-level and 2nd-level pages (so that we can hide the label for both pages).  For a 2nd-level page, if the field was called `child_ordering` it would suggest that it specifies the ordering for the children of the current page (ie the 3rd-level pages), when in fact it specifies the ordering of the siblings of the current page.  That's why I think `2nd_level_ordering` is a better name for ti in this context.